.github/workflows/ci.yml

```
@@ -145,3 +145,51 @@ jobs:
+            exit 1
+          fi
+      - name: Upload performance artifacts
+        if: always()
+        uses: actions/upload-artifact@v4
+        with:
+          name: performance-budgets
+          path: |
+            lighthouse-budget.json
+            tools/testing/performance-budget.json
+          if-no-files-found: ignore
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)